zipkin-query-service/config/query-cassandra.scala

```diff
@@ -37,18 +37,14 @@ if (username.isDefined && password.isDefined) {
 }
 
 val cluster = Factory.createClusterBuilder().build()
-val storeBuilder = Store.Builder(
-  new cassandra.SpanStoreBuilder(cluster),
-  new cassandra.DependencyStoreBuilder(cluster)
-)
-
-val loggerFactory = new LoggerFactory(
-  node = "",
-  level = Level.parse(logLevel),
-  handlers = List(ConsoleHandler())
-)
+val repository = new Repository(CassandraSpanStoreDefaults.KeyspaceName, cluster)
+val spanStore = new CassandraSpanStore(repository)
+val dependencies = new CassandraDependencyStore(repository)
 
 QueryServiceBuilder(
-  storeBuilder,
-  serverBuilder = ZipkinServerBuilder(serverPort, adminPort).loggers(List(loggerFactory))
+  "0.0.0.0:" + serverPort,
```

Comment:
[non-blocking] is there any significance to 0.0.0.0 in this case? is the intention the same as localhost?